src/test/java/com/commercetools/sync/products/helpers/variantreferenceresolver/withnestedattributes/WithCustomerReferencesTest.java

```diff
@@ -0,0 +1,74 @@
+package com.commercetools.sync.products.helpers.variantreferenceresolver.withnestedattributes;
+
+import com.commercetools.sync.products.ProductSyncOptions;
+import com.commercetools.sync.products.ProductSyncOptionsBuilder;
+import com.commercetools.sync.products.helpers.VariantReferenceResolver;
+import com.commercetools.sync.services.CategoryService;
+import com.commercetools.sync.services.ChannelService;
+import com.commercetools.sync.services.CustomerGroupService;
+import com.commercetools.sync.services.ProductService;
+import com.commercetools.sync.services.TypeService;
+import io.sphere.sdk.client.SphereClient;
+import io.sphere.sdk.json.SphereJsonUtils;
+import io.sphere.sdk.products.ProductVariantDraft;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static com.commercetools.sync.products.helpers.variantreferenceresolver.withnestedattributes.WithNoReferencesTest.RES_ROOT;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+
+class WithCustomerReferencesTest {
```

Comment:
Will you update that class when we add reference resolution for customer type? I guess also we don't have support customer-sync. What is our plan for those, maybe it could be useful to add something on to documentation as caveats like reference resolution for customer resource of product is not supported yet?

Reply:
Yes, I will add the caveat.

Reply:
bd6abbb